pkg/cmd/api/api.go

```diff
@@ -183,6 +185,10 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 			opts.RequestPath = args[0]
 			opts.RequestMethodPassed = c.Flags().Changed("method")
 
+			if runtime.GOOS == "windows" && filepath.IsAbs(opts.RequestPath) {
+				return fmt.Errorf("a Windows-style absolute path was passed. drop the leading slash or escape it")
```

Comment:
The logic looks good, but this error message means nothing to a regular user since:
1. they likely haven't supplied an absolute Windows-style path themselves;
2. they are likely unaware that mingw is doing automatic path translation in argument list;
3. they likely don't know the `//` syntax to escape the slash in the argument.

How about something like this? Feel free to suggest improvements:
> error: invalid API endpoint: C:\users\repos
> Your shell might be rewriting URL paths as filesystem paths. To avoid this, omit the leading slash from the endpoint argument.

Reply:
I see, @mislav. That makes sense. At the very least, we should show the user what exactly was passed to the command. I'll go with the error message you're suggesting.